imhotep/app.py

```diff
@@ -129,11 +136,22 @@ def invoke(self):
                     violating_lines)
                 for x in matching_numbers:
                     error_count += 1
+                    if error_count > max_errors:
+                        continue
                     reporter.report_line(
-                        repo.name, cinfo.origin, entry.result_filename,
+                        cinfo.origin, entry.result_filename,
                         x, pos_map[x], violations['%s' % x])
 
+                if error_count > max_errors \
```

Comment:
`max_errors` could be a parameter passed directly from the `imhotep` command when receiving a hook payload?